PAT docs for Konnect (#4499)
Kong/docs.konghq.com: app/konnect/getting-started/import.md

* {{site.konnect_saas}} [account credentials](/konnect/getting-started/access-account/).
* decK v1.12 or later [installed](/deck/latest/installation/).

## Generate a Personal Access Token

> To use decK to import entity configurations, it is recommended that you use a personal access token (PAT).

Reviewer: ```suggestion
To use decK to import entity configurations, we recommend that you use a personal access token (PAT).
```
